src/battleground/Battleground.js

```diff
@@ -55,7 +55,7 @@ const arenaWidth: {[ArenaType]: number} = {
 }
 
 const matchLengthForArena: {[ArenaType]: number} = {
-	DIRT: INTRO_LENGTH + 30*60,
+	DIRT: INTRO_LENGTH + 30*1,
```

Comment:
Is this supposed to be different from the rest?

Reply:
oops, no I just had that in there for testing so I didn't have to wait for the whole minute. Nice catch.